developer-docs-site/docs/nodes/ait/node-requirements.md

```diff
@@ -23,18 +23,49 @@ In test mode, you will be running a local network with one single node, and it s
 
 ## Hardware requirements
 
-For running an Aptos node we recommend the following hardware resources:
-
-  - **CPU**: 4 cores (Intel Xeon Skylake or newer).
+For running an Aptos **Validator node** we recommend the following hardware resources:
+
+  - **CPU**:
+      - 8 cores, 16 threads 
+      - 2.8GHz, or faster
+      - Intel Xeon Skylake or newer
+  - **Memory**: 32GiB RAM.
+
+Example machine types on various clouds:
+  - AWS
+      - c5.4xlarge
+      - c6i.4xlarge
+  - GCP
+      - c2-standard-16
+  - Azure
+      - Standard_B8ms
+
+For running an Aptos **Fullnode** we recommend the following hardware resources:
+
+  - **CPU**:
+      - 4 cores, 8 threads 
+      - 2.8GHz, or faster
+      - Intel Xeon Skylake or newer
   - **Memory**: 8GiB RAM.
```

Comment:
fullnodes won't keep up with validators with so little memory I assume.. thoughts @grao1991 ?

Reply:
I feel we should ask 16GiB.